lib/awesome_spawn.rb

```diff
@@ -110,8 +108,16 @@ def build_command_line(command, params = nil)
 
   private
 
-  def launch(command, spawn_options)
-    output, error, status = Open3.capture3(command, spawn_options)
+  def launch(env, command, spawn_options)
+    output, error, status = Open3.capture3(env, command, spawn_options)
     return output, error, status && status.exitstatus
   end
+
+  def parse_command_options(command, options)
+    options = options.dup
+    params  = options.delete(:params)
+    env = options.delete(:env) || {}
```

Comment:
Does having a blank hash here end up eliminating inherited environment variables?  i.e. what will this do:

```ruby
$ export SOME_VAR=1
$ irb
[1] pry(main)> require 'awesome_spawn'
=> true
[2] pry(main)> r = AwesomeSpawn.run!("echo $SOME_VAR")
=> #<AwesomeSpawn::CommandResult:0x007fcb460b8dd0 @exit_status=0>
[3] pry(main)> r.output
=> "1\n"
```